### PR TITLE
Add `BaseRecording.reset_times()` function

### PR DESCRIPTION
**Pull request by alejoe91**
This is a convenience function to discard time_vector information. It only acts in memory

### CODE REVIEW

**Review by samuelgarcia — APPROVED** [review]

**Comment by samuelgarcia** [comment]
OK for me.

**Comment by h-mayorquin** [comment]
LGTM as well.

Maybe we should add a note in the docstring that it only acts on the memory reference of the timestamps.

File: src/spikeinterface/core/baserecording.py [hunk 1]
```diff
@@ -495,6 +495,14 @@ def set_times(self, times, segment_index=None, with_warning=True):
                 "Use this carefully!"
             )
 
+    def reset_times(self):
+        """Reset times in-memory for all segments that have a time vector."""
```
Comment (alejoe91) [inline src/spikeinterface/core/baserecording.py:499]: @h-mayorquin isn't this `in-memory` enough? What would you recommend?

[the remark this answers is not in the document]
Comment (alejoe91) [reply src/spikeinterface/core/baserecording.py]: done!

**Pull request merged**